app/Models/Item.php

```diff
@@ -79,6 +85,7 @@ class Item extends MiscModel
         'size',
         'location.name',
         'character.name',
+        'item_id',
     ];
 
```

Comment:
I think `protected $sortable` is missing here for the items subpage. If you have multiple sub items, the "order by" (clicking on the header field name) won't work without it.